acl/types.proto

```diff
@@ -176,7 +176,8 @@ message EACLRecord {
     // Target subject's role class
     Role role = 1 [json_name="role"];
 
-    // List of public keys to identify target subject
+    // List of 25-byte accounts to identify target subjects.
+    // 33-byte public keys are also supported, however, they are deprecated and script hashes should be derived from them.
     repeated bytes keys = 2 [json_name="keys"];
```

Comment:
rename field to `accounts`?

Reply:
btw, still not sure if we can do this. we even have `[json_name="keys"]` notation in our protocol

Reply:
(but i would just rename it)

Reply:
i did not mean `json_name`. Protobuf field names are freely renamble. At the same time, divergence w/ json notation will come and this wouldnt be beautiful. So, since nobody cares much for now, lets keep